packages/shared/src/components/buttons/ToggleClickbaitShield.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { type ReactElement } from 'react';
+import { Button, ButtonSize, ButtonVariant } from './Button';
+import { ShieldCheckIcon, ShieldIcon, ShieldPlusIcon } from '../icons';
+import { useSettingsContext } from '../../contexts/SettingsContext';
+import { usePlusSubscription } from '../../hooks';
+import { SidebarSettingsFlags } from '../../graphql/settings';
+import { useLogContext } from '../../contexts/LogContext';
+import { LogEvent, TargetId } from '../../lib/log';
+import { useLazyModal } from '../../hooks/useLazyModal';
+import { SimpleTooltip } from '../tooltips';
+import { LazyModal } from '../modals/common/types';
+import { FilterMenuTitle } from '../filters/helpers';
+
+export const ToggleClickbaitShield = (): ReactElement => {
+  const { openModal } = useLazyModal();
+  const { isPlus } = usePlusSubscription();
+  const { logEvent } = useLogContext();
+  const { flags, updateFlag } = useSettingsContext();
+
+  if (!isPlus) {
+    return (
+      <SimpleTooltip
+        placement="bottom"
+        content="Enable Clickbait Shield to get clearer more informative titles"
```

Comment:
Where does this live, couldn't find it in design, maybe I overlooked it?

Reply:
https://dailydotdev.slack.com/archives/C0811D22VPG/p1733309611342949

Reply:
Asked product if we should add tooltips for them.

Reply:
Alright, just taxonomy addition needed I think as there's now 2 entry points to toggle from.